system/src/system_publish_vitals.cpp

```diff
@@ -6,11 +6,13 @@
 #include "logging.h"
 #include "system_cloud.h"
 #include "system_threading.h"
+#include "spark_wiring_timer.h"
+#include "../test/unit_tests/mock/mock_types.h"
 
 namespace
 {
 
-using namespace particle::system;
+//using namespace particle::system;
```

Comment:
I have fixed the headers that were needed to be changed because i could not compile, please advice if these should be removed from the PR.